building/export_plugin_from_eclipse.adoc

```diff
@@ -0,0 +1,37 @@
+= How to Export a plugin from Eclipse
+
+Exporting a plugin from inside eclipse is the 'quick and dirty' method to get an output binary for quick local tests. It does not generate an update site, and so this mechanism is only really suitable for local smoke-testing. 
```

Comment:
as long as the doc does not state this is going to generate the same as we actually release thats fine.

The *only* valid usecase for export from eclipse is really if you are doing something that does not build and just want to generate site for it anyway.